api-reference/rate_limiting.mdx

```diff
@@ -3,68 +3,19 @@ title: "Rate limiting"
 description: ""
 ---
 
-*Placeholder below*
+The Schematic API has a number of safeguards against bursts of requests to help maximize its stability. Users who send many requests in a quick succession might see error responses that show up as a `429`. We have a couple of mechanisms in the API, regardless of the environment, including:
 
-The Stripe API uses a number of safeguards against bursts of incoming traffic to help maximize its stability. Users who send many requests in quick succession might see error responses that show up as status code 429. We have several limiters in the API, including:
+1. A defined rate limit that allows up to 20 write operations per second and 20 read operations per second.
```

Comment:
This isn't strictly accurate but we don't have a way currently to guarantee a specific rate limit across the entire API, since it's done at a per-task level. Probably fine to just put roughs here, but just noting that it's not strictly correct

Reply:
ok - can modify if we feel more comfortable with no or different numbers, but will keep 20/20 for now.